src/utils/utils.ts

```diff
@@ -16,4 +16,22 @@ export function getRandomPairing(): NumPairs {
 
 export function getTestDate() {
     return new Date(Date.now() + 7 * 24 * 60 * 60 * 1000); //add 7 days to current time
-}
\ No newline at end of file
+}
+export function mapErrorCodeToMessage(code: string) {
+    switch (code) {
+        case "auth/email-already-in-use":
+            return "Email already exists.";
+        case "auth/invalid-email":
+            return "Invalid email.";
+        case "auth/weak-password":
+            return "Password is not strong enough.";
```

Comment:
What does firebase auth require in passwords, is it possible to be more specific here?

Reply:
I believe the only requirement is more than 6 characters.

Reply:
Ok, then could we say "Password length must be greater than 6 characters" or something along those lines instead of "Password is not strong enough"?